tasks.py

```diff
@@ -42,3 +42,26 @@ def imageid(ctx):
         image_name=DOCKER_IMAGE_NAME,
         format='{{.Id}}'
     ))
+
+def link_phutil_map(ctx):
+    ctx.run("ln -fs `pwd`/phutil_map /tmp/phutil_map")
+
+@task
+def build_test(ctx):
+    """Test phabricator extensions."""
+    link_phutil_map(ctx)
+    ctx.run("docker-compose -f docker-compose.test.yml build phabricator")
```

Comment:
ah, belay that; while they don't explicitly cover it in the docs, the Ubuntu 14.04 image does include compose 1.5.2. Ignore me.